Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Use an official Go runtime as a parent image
+FROM golang:1.21 as builder
+
+# Set the working directory inside the container
+WORKDIR /go/src/app
+
+# Copy the Go source code and .git directory into the container
+COPY . .
+
+# Build your Go app using the 'build' target in your Makefile
+RUN CGO_ENABLED=0 make build
+
+# Use a smaller base image to create a minimal final image
+FROM scratch
+
+# Set working directory
+WORKDIR /root/
+
+# Copy the binary from the builder stage
+COPY --from=builder /go/src/app/out/polycli .
+
+# Command to run the binary
+CMD ["./polycli"]\
```

Comment:
```suggestion
CMD ["polycli"]\
```

Reply:
tested this out. i think this needs extra configs? i'll just leave it as-is for now

![Uploading Screenshot 2024-01-23 at 5.56.31 PM.png…]()

Reply:
![Screenshot 2024-01-23 at 5 56 31 PM](https://github.com/maticnetwork/polygon-cli/assets/11021924/aeb54324-3b00-4fa2-bf15-3c4e6b332f3c)

Reply:
Resolved by 5ab29edb5003039ed0c1d92fc18826b6783dab47